### PR TITLE
Add Anyscale Integration

### PR DESCRIPTION
**Pull request by PineappleIOnic**
## What does this PR do?

Adds a write up for the Anyscale Integration in Appwrite AI docs

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
Yes

### CODE REVIEW

File: static/images/docs/ai/integrations/anyscale/demo.png
Comment (christyjacob4) [inline static/images/docs/ai/integrations/anyscale/demo.png:1]: The screenshot needs to be updated to "Anyscale Demo"

File: src/routes/docs/products/ai/integrations/anyscale/+page.markdoc [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+layout: article
+title: Integrating Anyscale
+description: Learn how to integrate Anyscale into your Appwrite project.
+difficulty: intermediate
+readtime: 15
+---
+
+The Anyscale API is a powerful tool for generating text using the leading open-source models. This tutorial will guide you through setting up the Anyscale API and integrating it into your Appwrite project.
+
+You'll create a simple function that takes a text prompt and generates a completion using Mistral's Mixtral 8x7B model. Then, using Appwrite functions, you'll make a user interface that allows users to input text and see the generated completion.
```
Comment (christyjacob4) [inline src/routes/docs/products/ai/integrations/anyscale/+page.markdoc:11]:
```suggestion
You'll create a simple function that takes a text prompt and generates a completion using Mistral's Mixtral 8x7B model. Then, using Appwrite functions, you'll create a UI that allows users to input text and see the generated completion.
```

File: src/routes/docs/products/ai/integrations/anyscale/+page.markdoc [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+
+{% only_light %}
+![Create function screen](/images/docs/functions/template.png)
+{% /only_light %}
+
+1. In the Appwrite Console's sidebar, click **Functions**.
+1. Click **Create function**.
+1. Under **Connect Git repository**, select your provider.
+1. After connecting to GitHub, under **Quick start**, select the **Node.js** starter template.
+1. In the **Variables** step, add `APPWRITE_BUCKET_ID` and `ANYSCALE_API_KEY`. Generate your AnyScale key [here](https://app.endpoints.anyscale.com/credentials) also tick the box to **Generate API key on completion**.
```
Comment (christyjacob4) [inline src/routes/docs/products/ai/integrations/anyscale/+page.markdoc:34]: why does it need a bucket and an Appwrite API key ?

File: static/images/docs/ai/integrations/anyscale/demo.png
Comment (PineappleIOnic) [reply static/images/docs/ai/integrations/anyscale/demo.png:1]: I'm not sure I follow here

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**